### PR TITLE
👉 RELEASE 1.19.0

### PR DESCRIPTION
**Pull request by henrypalacios**
# Summary
**Release v1.19.0**

---
:red_circle: DON'T SQUASH :red_circle:

## Main Features
- We added a popup to let the user know how to follow up on a pending order.
- Add support for the Goerli (GTH) test network.
- Using the new changes in [cow-sdk](https://github.com/cowprotocol/cow-sdk/pull/50) and [app-data](https://github.com/cowprotocol/app-data) relations Metadata schema.
- **Bug misc:** Invisible font fix in dark mode when wrapping

## Change Log
b8f3c4c00 increase padding-top to prevent overlapping (#819)
c4e1387f2 785/app data repo (#807)
f90478dab Fix traded amount on notifications (#817)
2b25e2fc7 804 - Fix ETH wrap font colour (#813)
4dac983b5 Adding FollowPendingTxPopup when TxSubmittedModal is close (#783)
31685920d Merge remote-tracking branch 'origin/main' into develop
e28097772 1.18.1
7e7e82679 Merge pull request #810 from cowprotocol/hotfix/1.18.1
deae87ceb Make the app resilient to a missing token list on localStorage (#805)
50596ae4e Goerli initial implementation (#559)
3fe62196a fix faq url (#803)
187a71972 Merge pull request #801 from cowprotocol/main
0bc6aa476 Add order execution progress bar (#797)
2b462efe8 Merge branch 'main' into develop
b786c1364 GA events 4 (#781)
c214b7c55 GA events for games and add to Metamask button (#780)
c07ca0e2f Added GA dimension for wallet name (#775)
a3d1d9777 Bumping cow-sdk to v0.0.15-RC.1 (#776)
69242e3da 332/Store slippage info on appData (#629)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   },
   "types": "./dist/widgets.d.ts",
   "private": true,
-  "version": "1.18.1",
+  "version": "1.19.0",
```
Comment (ramirotw) [inline package.json:27]: should we bump the version here or in a separate PR/commit?

**Comment by alfetopito** [comment]
Has this branch based off of `main`?
That should be the case for hotfixes, but this is a regular release, so should be based on `develop`

File: package.json (hunk 1 above)
Comment (henrypalacios) [reply package.json]: I did a [PR](https://github.com/cowprotocol/cowswap/pull/824) for this

**Comment by elena-zh** [comment]
I have verified the latest changes, and they LGTM. 
Another low-priority issue I have found is #827

**Review by alfetopito — APPROVED** [review]

**Comment by W3stside** [comment]
will add #815

**Review by ramirotw — APPROVED** [review]

**Pull request merged**